rrs/migrations/0004_maint_plan_default.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import settings
+
+
+def populate_plan(apps, schema_editor):
+    Release = apps.get_model('rrs', 'Release')
+    MaintenancePlan = apps.get_model('rrs', 'MaintenancePlan')
+    LayerBranch = apps.get_model('layerindex', 'LayerBranch')
+    MaintenancePlanLayerBranch = apps.get_model('rrs', 'MaintenancePlanLayerBranch')
+
+    if Release.objects.all().exists():
+        if not settings.CORE_LAYER_NAME:
+            raise Exception('Please set CORE_LAYER_NAME in settings.py')
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Using `Exception` and `BaseException` can make your code prone to errors and difficult to maintain. Instead, we recommend using one of the [Built-in Exceptions](https://docs.python.org/3/library/exceptions.html) or creating a custom exception class that is derived from `Exception` or one of its subclasses.



Similar issue at line number 19.